src/events-multiple/web/index.js

```diff
@@ -0,0 +1,31 @@
+// import { getIframeId, getWebfonts, reportClicks, resizeIframeHeight, onViewport } from '../../_shared/js/messages';
```

Comment:
could we clean up this file's commented lines? 🧹

Reply:
Indeed. I've gone through and got rid of a lot of comments. Thanks for catching it.